app/helpers/metric_chart_helper.rb

```diff
@@ -0,0 +1,51 @@
+module MetricChartHelper
+  def calculate_transaction_difference(current_metrics, previous_metrics, total_label)
+    @current_metrics = current_metrics
```

Comment:
Storing the current_metrics (and previous_metrics) is okay, but then they're only ever used to call .sum.  Maybe just stored current_metrics.sum so we don't have to do the calculation too much?

Reply:
Thanks @rossjones have updated.